kafka/pykafka/connection.py

```diff
@@ -74,10 +75,8 @@ def response(self):
         try:
             size = self._socket.recv(4)
             size = struct.unpack('!i', size)[0]
-            output = bytearray(size)
-            recvall_into(self._socket, output)
-            # TODO: Figure out if correlation ids are worth it
```

Comment:
This `TODO` is gone, did you ever look into how other drivers handle the coorelation ids?  This implementation doesn't get messages out of order, so matching them to the original request seemed needless.  Just wondering if you're thinking the same thing.